tests/contrib/aiomysql/test_aiomysql.py

```diff
@@ -133,3 +134,59 @@ async def test():
     )
     assert status == 0, err
     assert out == err == b""
+
+
+@pytest.mark.asyncio
+@pytest.mark.snapshot
+async def test_user_specified_service_v1(ddtrace_run_python_code_in_subprocess):
```

Comment:
I've taken a note to expanding the usage of this pattern, which can be parametrized, in a testing improvement follow-up action item.

Reply:
The service name is actually being checked via the `@pytest.mark.snapshot` decorator.  

When decorated, the function will gather a span and either compare it to an existing snapshot for the test (if the snapshot exists) or create a new snapshot file.  

In the case of this PR, you can see the snapshots added at the very end.